packages/orchestrator-ui-components/src/rtk/endpoints/forms.ts

```diff
@@ -0,0 +1,64 @@
+import { BaseQueryTypes, orchestratorApi } from '@/rtk';
+
+const PROCESS_ENDPOINT = 'processes';
```

Comment:
Are there scenarios where these would be changed so we should add them to the configuration? I guess not right?